docs/competing_risks.md

```diff
@@ -0,0 +1,278 @@
+	Andorra: Espot Zamora      0.990101  0.981595  0.975345  0.966500  ...  0.000727  0.000723  0.000718  0.000713
+	Angola: Lourenco           0.985230  0.972513  0.959758  0.947597  ...  0.025139  0.024493  0.023396  0.023232
+	                            ...       ...       ...       ...  ...       ...       ...       ...       ...
+	Venezuela: Nicolas Maduro  0.993150  0.986412  0.982183  0.975254  ...  0.000331  0.000329  0.000321  0.000313
+	Vietnam: Phu Trong         0.991983  0.982679  0.968412  0.959021  ...  0.002620  0.002586  0.002209  0.002123
+	Yemen: Houthi              0.992832  0.983635  0.950930  0.939746  ...  0.056119  0.055744  0.055367  0.054997
+	Zambia: Lungu              0.993150  0.986317  0.981646  0.973240  ...  0.008273  0.006682  0.005709  0.005411
+	Zimbabwe: Mnangagwa        0.993150  0.986421  0.981954  0.974727  ...  0.058133  0.057746  0.057357  0.056975
+	
+	[194 rows x 242 columns]
+	```
+
+##### Type of exit probabilities
+
+The same data can be used to train and forecast with the exit modeler, `LGBExitModeler`, where the terminal event type column name is passed as an argument to `exit_col`. The exit modeler produces forecasts for type of exit, conditional on :math:`t` being the last period observed. This is defined formally as
+
+.. math::
+   Pr(D_i=d_i|T_i= t,X_{i}).
+
+`LGBExitModeler` follows similar rules as `LGBSurvivalModeler` for selecting observations for training and forecasting. Except for training, where only observations with an observed exit in the desired time period are used.
+
+.. code-block:: python
+
+	# Obtain the probability of type of exit conditional on exit that period
+	exit_modeler = LGBExitModeler(data=mdata, exit_col = 'outcome')
+	exit_modeler.build_model(parallelize=False)
+	exit_modeler_forecasts = exit_modeler.forecast()
+	
+	# Set MultiIndex and column names
+	exit_modeler_forecasts = exit_modeler_forecasts.set_index('Future outcome', append = True, drop = True)
+	exit_modeler_forecasts = exit_modeler_forecasts.reindex(['Same government type','Change of government type','Dissolution','No exit'], axis=0,level=1)
+	exit_modeler_forecasts.columns = custom_forecast_headers
+	exit_modeler_forecasts
+	```
+	                                                         Jul 2020      Aug 2020  ...      Jul 2040      Aug 2040
+	country-leader            Future outcome                                         ...                            
+	Afghanistan: Ashraf Ghani Same government type       8.578190e-01  7.704733e-01  ...  7.762426e-01  7.535799e-01
+	                          Change of government type  1.419008e-01  2.289023e-01  ...  2.237574e-01  2.464201e-01
+	                          Dissolution                2.801606e-04  6.244381e-04  ...  9.282013e-16  9.760585e-16
+	                          No exit                    1.004800e-15  1.110349e-15  ...  9.282013e-16  9.760585e-16
```

Comment:
This value appearing as a possible outcome despite never occuring in training is a known issue. One workaround is to use one of the possible outcomes (e.g., "Same government type") as the value for non-last observations.

Reply:
Made adjustment. Not ideal. I prefer to change it back to "no_exit" once the bug is fixed.